packages/app/src/Element/Tabs.tsx

```diff
@@ -32,7 +32,7 @@ const Tabs = ({ tabs, tab, setTab }: TabsProps) => {
   return (
     <div className="tabs" ref={horizontalScroll}>
       {tabs.map(t => (
-        <TabElement tab={tab} setTab={setTab} t={t} />
+        <TabElement key={t.text} tab={tab} setTab={setTab} t={t} />
```

Comment:
There is `t.value` also?

Reply:
`value` is the index of the tab so i suppose its actually meant to be unique unlike text